src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```
@@ -581,6 +581,9 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkGitHubVersion)" PrivateAssets="All" />
+    <PackageReference Include="System.Text.Encodings.Web">
+      <Version>$(SystemTextEncodingsWebVersion)</Version>
+    </PackageReference>
```

NIT: Just prefering single line reference.
```suggestion
    <PackageReference Include="System.Text.Encodings.Web" Version="$(SystemTextEncodingsWebVersion)" />
```

---

I tried it before, this does not work in NetFx project.